util/memory/tracker_test.go

```diff
@@ -430,29 +429,23 @@ func (s *testSuite) TestFormatBytesWithPrune(c *C) {
 		{"9.15999984741211 MB", "9.16 MB"},
 	}
 	for _, ca := range cases {
-		b, err := s.parseByte(ca.b)
-		c.Assert(err, IsNil)
+		b, err := parseByte(ca.b)
+		require.NoError(t, err)
 		result := FormatBytes(b)
-		c.Assert(result, Equals, ca.s, Commentf("input: %v", ca.b))
+		require.Equal(t, ca.s, result)
```

Comment:
Use `Equalf` to contain `fmt.Printf("input: %v\n", ca.b)` maybe.

Reply:
Then you can remove `fmt.Print`, which will output something whatever the result is. It is annoying.

Reply:
I regarded `Commentf` as `fmt.Printf` previously, I have just check the document of `go-check` and found that it won't print until the assert is false. Sorry for my misunderstanding.

Reply:
Test fails in `tidbTestSerialSuite.TestTLS` twice, I try to remove `t.Parallel()` in concurrent test `TestConsume` to eliminate it.